adit/upload/models.py

```diff
@@ -0,0 +1,30 @@
+from typing import TYPE_CHECKING
+
+from django.db import models
+from django.urls import reverse
+
+from adit.core.models import DicomAppSettings, TransferJob, TransferTask
+
+if TYPE_CHECKING:
+    from django.db.models.manager import RelatedManager
+# Create your models here.
+
+
+class UploadSettings(DicomAppSettings):
+    class Meta:
+        verbose_name_plural = "Upload settings"
+
+
+class UploadJob(TransferJob):
```

Comment:
I already forgot why exactly we need a model. Was it for logging purposes? Or to provide a ModelForm (which would not be a good reason)? And do we update the status of this model correctly? Do we have a view where we can view all upload jobs and tasks?